src/sutta_processor/shared/config.py

```diff
@@ -81,6 +81,9 @@ def from_yaml(cls, f_pth: Union[str, Path] = None) -> "ExcludeRepo":
 
 @attr.s(frozen=True, auto_attribs=True)
 class Config:
+    # Not setting default so that exclude_dirs must be included
+    exclude_dirs: List[str] = attr.ib()
```

Comment:
Since this new attribute doesn't have a default, it had to be above the attributes with defaults.  This is an `attrs` requirement.